components/StorageFileList.vue

```diff
@@ -34,14 +35,26 @@ const sortFiles = () => {
     />
     <template v-else>
       <div class="files__menu">
-        <SortFileList @set-sort-options="sortFiles" />
-        <LayoutSwitcher />
+        <!-- v-if="selectedFiles" added to get rid of hydration errors caused by useSelectedFiles composable -->
+        <MultipleFilesMenu
+          v-if="selectedFiles"
+          class="files__menu--multiple"
+          :file-list="fileList"
+          @files-action="updateList"
+        />
```

Comment:
The useSelectedFiles composable was causing problems when I wanted to refresh the page. I remember you send me some link to github that in higher Nuxt version is a problem. I don't understand why but this solved the problem.